packages/ecs/src/lib/Entity.ts

```diff
@@ -10,22 +10,26 @@ import defaultWorld from './defaultWorld';
  * The Entity representation, which can carry mulitple components and appear in {@linkcode Query}s.
  */
 export default class Entity {
+	public static isDestroyed( entity: Entity ): boolean {
```

Comment:
Please move this down past the constructor.

Reply:
Actually, please remove this method, as we already have a method with this functionality.